core/src/qos_service.rs

```diff
@@ -260,6 +301,24 @@ struct QosServiceMetrics {
 
     // number of transactions to be queued for retry due to its account data limits
     retried_txs_per_account_data_limit_count: AtomicU64,
+
+    // accumulated estimated signature Compute Unites to be packed into block
+    estimated_signature_units: AtomicU64,
+
+    // accumulated estimated write locks Compute Units to be packed into block
+    estimated_write_lock_units: AtomicU64,
+
+    // accumulated estimated instructino data Compute Units to be packed into block
+    estimated_data_bytes_units: AtomicU64,
+
+    // accumulated estimated program Compute Units to be packed into block
+    estimated_execute_units: AtomicU64,
+
+    // accumulated actual program Compute Units that have been packed into block
+    actual_execute_units: AtomicU64,
```

Comment:
Can we start calling these `_cu` instead of ambiguous `_units`?

Reply:
Yeah, time of `cu` has arrived 😄